profile-patches/ibm-cloud-managed/02-deployment.yaml.patch

```diff
@@ -3,6 +3,7 @@
   value:
     config.openshift.io/inject-proxy: ingress-operator
     include.release.openshift.io/ibm-cloud-managed: "true"
+    capability.openshift.io/name: Ingress
```

Comment:
Please alphabetize.  (It looks like the patched file is alphabetized though.)

Reply:
Done.